tests/tests-customers-db.php

```diff
@@ -141,8 +151,7 @@ public function test_installed() {
 	public function test_get_by() {
 		$customer = edd_get_customer_by( 'email', 'testadmin@domain.com' );
 
-		$this->assertInternalType( 'object', $customer );
-		$this->assertObjectHasAttribute( 'email', $customer );
+		$this->assertGreaterThan( 0, $customer->id );
```

Comment:
Shouldn't we be testing that a specific customer was returned since we're passing a specific email? ;-)